dir.props

```diff
@@ -555,6 +563,14 @@
     <PackagesBasePath Condition="'$(PackagesBasePath)'==''">$(BinDir)$(OSPlatformConfig)</PackagesBasePath>
     <PackageOutputPath Condition="'$(PackageOutputPath)'==''">$(PackageOutputRoot)$(ConfigurationGroup)/</PackageOutputPath>
     <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)symbols/</SymbolPackageOutputPath>
+  
+    <!-- defaults for BuildTestsAgainstPackages -->
+    <!-- PackageInstallPath is where the locally built packages are installed -->
+    <PackageInstallPath>$(BinDir)localpackages</PackageInstallPath>
```

Comment:
@karajas The default in buildtools is `localpkg`, as declared at https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets#L69.  Is there a reason to use `localpackages` here instead?